editoast/src/views/mod.rs

```diff
@@ -16,25 +16,38 @@ pub mod study;
 pub mod timetable;
 pub mod train_schedule;
 
-use self::openapi::OpenApiMerger;
+use self::openapi::{merge_path_items, OpenApiMerger, Routes};
 use crate::client::get_app_version;
 use crate::core::version::CoreVersionRequest;
 use crate::core::{AsCoreRequest, CoreClient};
 use crate::error::Result;
 use crate::map::redis_utils::RedisClient;
-use crate::DbPool;
+use crate::{schemas, DbPool};
 use actix_web::dev::HttpServiceFactory;
 use actix_web::web::{Data, Json};
 use actix_web::{get, services};
 use diesel::sql_query;
 use diesel_async::RunQueryDsl;
 use redis::cmd;
 use serde_derive::{Deserialize, Serialize};
-use utoipa::{OpenApi, ToSchema};
+use utoipa::openapi::{Array, Object};
+use utoipa::{
+    openapi::{schema::AnyOf, AllOf, OneOf, RefOr, Schema},
+    OpenApi, ToSchema,
+};
+
+fn routes_v2() -> Routes<impl HttpServiceFactory> {
```

Comment:
We could add a comment to explain that this is temporarily used for the migration to utoipa.